Markdown/Markdown.sublime-syntax

```diff
@@ -1292,66 +1437,144 @@ contexts:
       escape_captures:
-        0: meta.code-fence.definition.end.shell-script.markdown-gfm
+        0: meta.code-fence.definition.end.matlab.markdown-gfm
         1: punctuation.definition.raw.code-fence.end.markdown
+
+  fenced-objc:
+    - match: |-
+         (?x)
+          {{fenced_code_block_start}}
+          ((?i:objc|obj-c|objectivec|objective-c))
+          {{fenced_code_block_trailing_infostring_characters}}
+      captures:
+        0: meta.code-fence.definition.begin.objc.markdown-gfm
+        2: punctuation.definition.raw.code-fence.begin.markdown
+        5: constant.other.language-name.markdown
+      embed: scope:source.objc
+      embed_scope: markup.raw.code-fence.objc.markdown-gfm
+      escape: '{{code_fence_escape}}'
+      escape_captures:
+        0: meta.code-fence.definition.end.objc.markdown-gfm
+        1: punctuation.definition.raw.code-fence.end.markdown
+
+  fenced-objcpp:
+    - match: |-
+         (?x)
+          {{fenced_code_block_start}}
+          ((?i:objc\+\+|obj-c\+\+|objectivec\+\+|objective-c\+\+))
+          {{fenced_code_block_trailing_infostring_characters}}
+      captures:
+        0: meta.code-fence.definition.begin.objc++.markdown-gfm
+        2: punctuation.definition.raw.code-fence.begin.markdown
+        5: constant.other.language-name.markdown
+      embed: scope:source.objc++
+      embed_scope: markup.raw.code-fence.objc++.markdown-gfm
+      escape: '{{code_fence_escape}}'
+      escape_captures:
+        0: meta.code-fence.definition.end.objc++.markdown-gfm
+        1: punctuation.definition.raw.code-fence.end.markdown
+
+  fenced-ocaml:
+    - match: |-
+         (?x)
+          {{fenced_code_block_start}}
+          ((?i:ocaml))
+          {{fenced_code_block_trailing_infostring_characters}}
+      captures:
+        0: meta.code-fence.definition.begin.ocaml.markdown-gfm
+        2: punctuation.definition.raw.code-fence.begin.markdown
+        5: constant.other.language-name.markdown
+      embed: scope:source.ocaml
+      embed_scope: markup.raw.code-fence.ocaml.markdown-gfm
+      escape: '{{code_fence_escape}}'
+      escape_captures:
+        0: meta.code-fence.definition.end.ocaml.markdown-gfm
+        1: punctuation.definition.raw.code-fence.end.markdown
+
+  fenced-perl:
+    - match: |-
+         (?x)
+          {{fenced_code_block_start}}
+          ((?i:perl))
+          {{fenced_code_block_trailing_infostring_characters}}
+      captures:
+        0: meta.code-fence.definition.begin.perl.markdown-gfm
+        2: punctuation.definition.raw.code-fence.begin.markdown
+        5: constant.other.language-name.markdown
+      embed: scope:source.perl
+      embed_scope: markup.raw.code-fence.perl.markdown-gfm
+      escape: '{{code_fence_escape}}'
+      escape_captures:
+        0: meta.code-fence.definition.end.perl.markdown-gfm
+        1: punctuation.definition.raw.code-fence.end.markdown
+
+  fenced-php:
     - match: |-
          (?x)
           {{fenced_code_block_start}}
```

Comment:
I know this is out of scope. Could we remove the `inc` from the next line `((?i:php|inc))` (in this PR) incidentally?

ref: https://github.com/sublimehq/Packages/pull/1550

Reply:
The Markdown code fence aliases are based on what GitHub handles: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L423

Reply:
Interesting...